### DISCUSSION TITLE
Can't seem to get privateGPT to work on Ubuntu.

### DISCUSSION THREAD

**Discussion by Oshirowanen** [imartinez/privateGPT#995] [Q&A]
Trying to get PrivateGPT working on Ubuntu 22.04 (I've also tired it on 18.04) but I keep getting a ton of errors.  Does anyone have a comprehensive guide on how to get this to work on Ubuntu?  The errors I am getting are dependency and version issues.  such as the wrong version of PIP, torch, python, and many many other missing dependencies.   I have tried to fix the errors one at a time as they happen, but I seem to be getting nowhere.

The errors happen when I try to run "pip3 install -r requirements.txt"

Can anyone help who has got this successfully working on Ubuntu 18.04 or 22.04?

**Comment by mirislam** [comment]
What errors are you getting? Can you post the output?

**Comment by PasunuriSrinidhi** [comment]
I am also unable to install all the requirements, can you please help me to resolve the following errors 

ERROR: Could not find a version that satisfies the requirement gpt4all==1.0.8 (from versions: 0.1.5, 0.1.6, 0.1.7)
ERROR: No matching distribution found for gpt4all==1.0.8
ubuntu@os

**Reply by Rakly3** [reply]
pip install gpt4all==1.0.8

You're likely going to get more, simply do _pip install_ for each of them

**Comment by muksho** [comment]
This is working fine for me in Ubuntu.
